eng/pipelines/patch-release-prepare.yml

```
@@ -5,52 +5,70 @@
 #
 # What this script does:
 #   - Checks out a new release branch, 'release/patch/$currentDate'
+#   - Pushes the release branch to the repo
 #   - Runs the script to generate patches for automatic releases
 #   - Create a PR with the patches prepared
 
 trigger: none
 pr: none
 
-parameters:
-  - name: TestPipeline
-    type: boolean
-    default: false
+extends:
+  template: /eng/pipelines/templates/stages/1es-redirect.yml
+  parameters:
+    stages:
+      - stage: PreparePatchRelease
+        displayName: Prepare Patch Release
+        variables:
+          - template: /eng/pipelines/templates/variables/image.yml
+        jobs:
+          - job: PreparePatchRelease
+            displayName: 'Prepare Patch Release'
+            timeoutInMinutes: 30
+            pool:
+              name: $(LINUXPOOL)
+              image: $(LINUXVMIMAGE)
+              os: $(LINUXOS)
```

This one specifically needs to be hardcoded due to 1es template limitations.

```suggestion
              os: linux
```